modules/nf-core/bbmap/bbmerge/main.nf

```diff
@@ -0,0 +1,59 @@
+process BBMAP_BBMERGE {
+    tag "$meta.id"
+    label 'process_single'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.06--h92535d8_0':
+        'biocontainers/bbmap:39.06--h92535d8_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_merged.fastq"), emit: merged
+    tuple val(meta), path("*_unmerged.fastq"), emit: unmerged
+    tuple val(meta), path("*_ihist.txt"), emit: ihist
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    maxmem=\$(echo \"$task.memory\"| sed 's/ GB/g/g')
+    bbmerge.sh \\
+        -Xmx\$maxmem \\
+        in1=${reads[0]} \\
+        in2=${reads[1]} \\
+        out=${prefix}_merged.fastq \\
+        outu1=${prefix}_1_unmerged.fastq \\
+        outu2=${prefix}_2_unmerged.fastq \\
```

Comment:
It would be preferable as good data stewards to have the module outputting compressed fastqs.

Do you think you could do it? Maybe it could be asked for help in the `nf-core` slack channel.

If none of the above work, I am fine with merging it as is, but also opening an issue in the github `nf-core/modules` about this so someone else can take care of it once this is merged.